src/components/EmojiPicker/EmojiPicker.js

```diff
@@ -72,7 +72,8 @@ const EmojiPicker = forwardRef((props, ref) => {
         const anchorOriginValue = anchorOrigin || DEFAULT_ANCHOR_ORIGIN;
 
         calculateAnchorPosition(emojiPopoverAnchor.current, anchorOriginValue).then((value) => {
-            onWillShow();
+            // eslint-disable-next-line es/no-optional-chaining
```

Comment:
I'm inclined to let it slide because this will be migrated to TS soon, but we shouldn't use optional chaining in JS files